code/datums/supply_packs/security.dm

```diff
@@ -0,0 +1,406 @@
+					/obj/item/weapon/gun/energy/smalldisintegrator)
+	cost = 60
+	containertype = /obj/structure/closet/crate/secure/ayy_mdf
+	containername = "MDF standard weapons crate"
+	one_access = list(access_armory, access_mothership_military)
+	group = "Security"
+	hidden = 1
+	containsdesc = "Surplus from the nearest mothership, in case of human uprising."
+
+/datum/supply_packs/smartlaser
+	name = "Smart laser guns"
+	contains = list(/obj/item/weapon/gun/energy/laser/smart,
+					/obj/item/weapon/gun/energy/laser/smart,
+					/obj/item/weapon/gun/energy/laser/smart,
+					/obj/item/weapon/gun/energy/laser/smart)
+	cost = 60
+	containertype = /obj/structure/closet/crate/secure/weapon
+	containername = "smart laser guns crate"
+	access = list(access_armory)
+	group = "Security"
+	containsdesc = "Four high tech laser guns from Central Command. Handle with care."
+
+/datum/supply_packs/eweapons
+	name = "Incendiary weapons"
+	contains = list(/obj/item/weapon/gun/projectile/flamethrower/full,
+					/obj/item/weapon/tank/plasma,
+					/obj/item/weapon/tank/plasma,
+					/obj/item/weapon/tank/plasma,
+					/obj/item/weapon/grenade/chem_grenade/incendiary,
+					/obj/item/weapon/grenade/chem_grenade/incendiary,
+					/obj/item/weapon/grenade/chem_grenade/incendiary)
+	cost = 25
+	containertype = /obj/structure/closet/crate/secure/weapon
+	containername = "incendiary weapons crate"
+	access = list(access_heads)
+	group = "Security"
+	containsdesc = "A flamethrower and three incendiary grenades, perfect for dealing with alien infestations aboard your station."
+
+/datum/supply_packs/armor
+	name = "Armor"
+	contains = list(/obj/item/clothing/head/helmet/tactical/sec,
+					/obj/item/clothing/head/helmet/tactical/sec,
+					/obj/item/clothing/suit/armor/bulletproof,
+					/obj/item/clothing/suit/armor/bulletproof,
+					/obj/item/clothing/suit/armor/vest,
+					/obj/item/clothing/suit/armor/vest)
+	cost = 20
+	containertype = /obj/structure/closet/crate/secure/basic
+	containername = "armor crate"
+	access = list(access_security)
+	group = "Security"
+	containsdesc = "A crate with two bulletproof vests, two general armor vests, and two tactical helmets."
+
+/datum/supply_packs/greyarmor
+	name = "MDF surplus standard armor"
+	contains = list(/obj/item/clothing/suit/armor/mothership,
+					/obj/item/clothing/suit/armor/mothership,
+					/obj/item/clothing/under/grey/grey_soldier,
+					/obj/item/clothing/under/grey/grey_soldier,
+					/obj/item/clothing/shoes/jackboots/mothership,
+					/obj/item/clothing/shoes/jackboots/mothership,
+					/obj/item/clothing/head/helmet/mothership,
+					/obj/item/clothing/head/helmet/mothership)
+	cost = 60
+	containertype = /obj/structure/closet/secure_closet/ayy2
+	containername = "MDF standard armor locker"
+	one_access = list(access_security, access_mothership_military)
+	group = "Security"
+	contraband = 1
+	containsdesc = "Surplus armor from the mothership. Not fitted for human consumption."
```

Comment:
Human consumption for non-edibles is a weird turn of phrase. Maybe just "Won't fit humans"?

Reply:
calling @Optimism333 for their thoughts on this (and all the other grey crate descriptions...)

Reply:
MDF armor DOES fit humans, in fact, all mothership clothing fits humans (the uniforms, explorer armor, MDF armor), it's the other species and fatties they don't account for. It should also be noted that ayy rigsuits only fit ayys, so something like "designed for only the slimmest of grey bodies" would likely be more appropriate.